docs/administration/projects/node-execution/ssh.md

```diff
@@ -571,17 +571,43 @@ environment.
 
 ### Passing environment variables through remote command
 
-To pass environment variables through remote command
-dispatches, it is required to properly configure the SSH server on the
-remote end. See the `AcceptEnv` directive in the "sshd_config(5)"
-manual page for instructions.
-
-Use a wild card pattern to permit `RD_` prefixed variables to provide
-open access to Rundeck generated environment variables.
+ - To pass environment variables through remote command
+    dispatches, it is required to properly configure the SSH server on the
+    remote end. See the `AcceptEnv` directive in the "sshd_config(5)"
+    manual page for instructions.
+    
+    Use a wild card pattern to permit `RD_` prefixed variables to provide
+    open access to Rundeck generated environment variables.
 
 Example in sshd_config:
 
     # pass Rundeck variables
     AcceptEnv RD_*
 
 [ssh]: https://en.wikipedia.org/wiki/Secure_Shell
+
+
+- To pass environment variables directly throught the command it self, you 
+can also make use of these node attributes to stablish the configuration by
+node
+  - ssh-variable-export-pattern : [STRING] Allows to explictly set the export 
+  sintaxis, it must have {key} and {value}.
```

Comment:
`sintaxis` is not correct english, either "syntaxes", or "pattern", i think pattern is better
```suggestion
  pattern, it must have {key} and {value}.
```